scripts/places_missing_provider_data.py

```diff
@@ -69,3 +69,42 @@ def _is_cache_missing_required_keys(required_keys, cache_for_place_id):
         factual_child = cache_for_place_id['factual']
         if _KEY_WEBSITE not in factual_child: return True
     return False
+
+import pprint
+def calculate_crawled_provider_stats(center, radius_km):
+    statusTable = _firebase.database().child(venuesTable).child("status").get().val()
+
+    # Fetch placeIDs to check
+    location_table = _firebase.database().child(locationsTable).get().val()
+    placeIDs = geo.get_place_ids_in_radius(center, radius_km, location_table)
```

Comment:
nit: if you don't pass in a location table, geo will get one for you so no need to do that here.